fabric-transfer-api-v1/src/client/java/net/fabricmc/fabric/api/transfer/v1/client/fluid/FluidVariantRenderHandler.java

```diff
@@ -73,7 +73,7 @@ default Sprite[] getSprites(FluidVariant fluidVariant) {
 	}
 
 	/**
-	 * Return the color to use when rendering {@linkplain #getSprite the sprite} of this fluid variant.
+	 * Return the color to use when rendering {@linkplain FluidVariantRendering#getSprite the sprite} of this fluid variant.
```

Comment:
This should actually link to `#getSprites`. (`#getSprite` was superseded by the plural version, and eventually removed).

Reply:
@Technici4n If that's the case, it should be `@Deprecated(forRemoval=true)`d, right?

Reply:
No. I meant `FluidVariantRenderHandler#getSprite` which doesn't exist anymore, and where this used to link. The color applies to both sprites, so it's more correct to link to `#getSprites`.